packages/url_launcher/url_launcher_web/test/url_launcher_web_test.dart

```diff
@@ -54,6 +54,17 @@ void main() {
       expect(newWindow.opener, equals(html.window));
     });
 
+    test('the url is opened in the same window', () {
+      // Simulate the navigator is in standalone mode on iOS devices.
+      // https://developer.mozilla.org/en-US/docs/Web/API/Navigator
+      navigator.standalone = true;
```

Comment:
Since this is modifying global state from the test, please reset it after the test is done... Something like:

```
final originalStandalone = navigator.standalone;
... your test
navigator.standalone = originalStandalone;
```

Reply:
Done!